packages/webview_flutter/webview_flutter_platform_interface/test/src/method_channel/webview_method_channel_test.dart

```diff
@@ -0,0 +1,458 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import 'package:webview_flutter_platform_interface/src/method_channel/webview_method_channel.dart';
+import 'package:webview_flutter_platform_interface/src/utils/javascript_channel_registry.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('$MethodChannelWebViewPlatform', () {
```

Comment:
I don't know why the plugins repo does so much of this; tests are already grouped by the file they are in. I'd rather just not have a group wrapping all tests. (Logic groups of sets of tests within the file are fine, it's just these top-level ones that seem like noise.)

Reply:
I have updated the description so the grouping makes more sense. At the moment there are 2 groups:

- One that tests the API on the `plugin.flutter.io/webview_<channel_id>` method channel;
- A second group that tests the API on the `plugins.flutter.io/cookie_manager` method channel.